packages/web/pages/docs/lists.mdx

```diff
@@ -0,0 +1,66 @@
+import Link from "next/link";
+import Image from "next/image";
+
+# Mailing Lists
+
+Mailing Lists make sure your users only get the emails they want. With Lists, you can include a link for users to update their email preferences in your templates by adding a magic link to your mailing template. Mailing will keep track of what their preferences are and show them the option to unsubscribe or resubscribe to any list that they’ve ever been sent from before.
+
+We hope you and your users will enjoy using Lists!
+
+// This paid feature is part of [Mailing Platform](/docs/platform)
```

Comment:
@psugihara is this supposed to be a comment?  It appears on the site with // in front of the text because the file is markdown

Reply:
yep, supposed to appear there as a note